torch/csrc/lazy/core/config.h

```diff
@@ -8,6 +8,8 @@ C10_DECLARE_bool(torch_lazy_all_numbers_special_scalars);
 C10_DECLARE_bool(torch_lazy_param_aliasing);
 C10_DECLARE_bool(torch_lazy_reuse_ir);
 C10_DECLARE_bool(torch_lazy_use_thread_pool);
+C10_DECLARE_bool(torch_lazy_enable_compilation_cache);
```

Comment:
where is this used?

Reply:
Compilation cache is not currently used - I added this for symmetry - but I'm also happy to remove it, or add a python binding for XLA.

Reply:
@wconstab removed the unused code

Reply:
Ping @wconstab

Reply:
sry just back from pto. will also cc @JackCaoG for a review.